source/NVDAObjects/window/__init__.py

```diff
@@ -169,6 +169,9 @@ def __init__(self,windowHandle=None):
 		self.windowHandle=windowHandle
 		super(Window,self).__init__()
 
+	def _get_uniqueID(self):
+		return self.windowHandle

```

Comment:
I wonder how often window handles get reused.

```suggestion
		return (self.processID, self.windowHandle)
```